src/graph.ts

```diff
@@ -552,8 +630,7 @@ export class FactoryGraph {
      */
     createContainer(item: Item): ContainerNode {
         const containers = this.getContainers(item)
-        const name = item.name + " C" + containers.size
-        const container = new ContainerNode(name, item, this, 1.0)
+        const container = new ContainerNode("C" + containers.size, item, this, 1.0)
```

Comment:
I would use a template literal here. Makes the absence of whitespaces immediately apparent.
```suggestion
        const container = new ContainerNode(`C${containers.size}`, item, this, 1.0)
```

Reply:
Fixed in https://github.com/lgfrbcsgo/du-factory-generator/commit/e3612727cf42e8f92e923180d0c72033bd0a1513